app/models/event.rb

```diff
@@ -268,6 +269,19 @@ def time
     event_schedules.find_by(schedule_id: selected_schedule_id).try(:start_time)
   end
 
+  ##
+  # Returns true or false, if the event is already over or not
+  #
+  # ====Returns
+  # * +true+ -> If the event is over
+  # * +false+ -> If the event is not over yet
+  def ended?
+    return false unless selected_schedule_id
```

Comment:
what about

```ruby
return false unless selected_schedule_id || event_schedule
```

and remove [line 281](https://github.com/openSUSE/osem/pull/1100/files#diff-6ecd61439b970b0e5ae49be738861446R281)